examples/deep_clone_schema.py

```diff
@@ -32,7 +32,7 @@
 
 # COMMAND ----------
 
-# %pip install dbl-discoverx==0.0.7
+# %pip install dbl-discoverx==0.0.8
```

Comment:
I agree with Sourav's comment to move this before the widget section. Shouldn't we also uncomment the restartPython part? If not I would remove it but I think it's good to keep it.